packages/forms/test/form_group_spec.ts

```diff
@@ -307,6 +358,16 @@ describe('FormGroup', () => {
         g.setValue({'one': 'one', 'two': 'two'});
         expect(logger).toEqual(['control1', 'control2', 'group', 'form']);
       });
+
+      it('should not emit events on the parent when called with `emitEvent: false`', () => {
+        form.valueChanges.subscribe(() => logger.push('form value change'));
+        g.valueChanges.subscribe(() => logger.push('group value change'));
+        form.statusChanges.subscribe(() => logger.push('form status change'));
+        g.statusChanges.subscribe(() => logger.push('group status change'));
+
+        g.addControl('three', new FormControl(5, Validators.min(10)), {emitEvent: false});
```

Comment:
It looks like validator is not needed in this case too:
```suggestion
        g.addControl('three', new FormControl(5), {emitEvent: false});
```